packages/client/components/DiscussionThreadInput.tsx

```diff
@@ -226,7 +226,7 @@ const DiscussionThreadInput = forwardRef((props: Props, ref: any) => {
       if (!editorEl || editorEl.type !== 'textarea') return
       const {value} = editorEl
       if (!ensureHasText(value)) return
-      addComment(convertToTaskContent(value))
+      addComment(convertAndroidValueToTaskContent(value))
```

Comment:
- Actually, this fix is only meant for Android devices since they are the only ones experiencing this issue.
- We found out that this comment is already being called within a conditional statement in line 225 of the `DiscussionThreadInput.tsx` file checking if it is an Android device, so we guessed this function was originally built only for cases where Android devices are used.